duckduckgo/Android
# Feature/david/contrast colors change

Task/Issue URL: https://app.asana.com/0/715106103902962/874107794298116
Product Review URL: https://app.asana.com/0/1142021229838617/1176459710279645 

**Description**:
In a handful of scenarios in our Android app we fail to meet WCAG (Web Content Accessibility Guidelines https://www.w3.org/WAI/standards-guidelines/wcag/) standards. This results in a problematic user experience for people with visual impairments and suboptimal experience for everyone else.

This PR fixes those issues, using new colors that meet the above mentioned guidelines

**Steps to test this PR**:
1. Open the app in Light Mode and Dark Mode
2. Navigate through the app and check the new colors as per https://www.figma.com/file/DTS9R8KSj29PANxUUCdfWJ/Optimize-Themes?node-id=0%3A1

---
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
